scripts/monitoring/cron-send-etcd-status.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# setting up the certificate and the key
+CERT=`ls /etc/openshift/node/system*node*.crt`
+KEY=`ls /etc/openshift/node/system*node*.key`
+
+# querying the health ping
+wget -q -O- --no-check-certificate --certificate=${CERT} --private-key=${KEY} https://0.0.0.0:10250/healthz/ping |grep -c "ok" >/dev/null
```

Comment:
It's more than that, though. You're running this on all of the nodes. And you're binding in the node's cert. You're also hitting 0.0.0.0, which is "all local interfaces."

That's why I said this seems to be monitoring the kubelet and not etcd.